addons/tutorial_03_multi_hop/manifest.json

```diff
@@ -13,7 +13,7 @@
     "title": "Using multi-hop VPN",
     "settings_rollback_needed": true,
     "subtitle": "Follow this walkthrough to learn how to use multi-hop VPN.",
-    "completion_message": "You’ve turned on multi-hop VPN and changed your exit location. Would you like to see more tips and tricks?",
+    "completion_message": "You’ve learned how to use multi-hop VPN and change your exit location. Would you like to learn more tips and tricks?",
```

Comment:
@bakulf 
Can we use a different ID in these manifests?

Reply:
oh no! Big problem here... not in the current implementation! Same for title, subtitle and so on. We need to find a solution. Follow up.

Reply:
In this scenario, since this tutorial isn't public yet, would keeping the id the same be okay?

Reply:
It's not important if it's public, it matters if it's been already exposed to localization (I think it is).

As I said, I build the automation to invalidate existing translations if the source string changes, but we should figure out a way to work around this for the future.